[fx][Transformer] Add override for call_function

jfix71 opened the pull request:
Differential Revision: D29151191

This ensures that if a function was wrap'd before symbolic tracing + being passed into the transformer then it will still be wrapped.

jamesr66a reviewed (commented):
Looks good to me but the implementation is commented out

jfix71 committed:
[fx][Transformer] Add override for call_function (#60057)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/60057

This ensures that if a function was `wrap`'d before symbolic tracing + being passed into the transformer then it will still be wrapped.

Test Plan: Added test to `test_fx.py`

Differential Revision: D29151191

fbshipit-source-id: d56efe5b974ad87cd930fe346a1c394ac67c5c34
